pdns/histogram.hh

```diff
@@ -35,8 +36,10 @@ namespace pdns
 // By convention, we are using microsecond units
 struct Bucket
 {
-  const std::string d_name;
-  const uint64_t d_boundary{0};
+  Bucket(std::string name, uint64_t boundary, uint64_t val) :
+    d_name(std::move(name)), d_boundary(boundary), d_count(val) {}
+  std::string d_name;
+  uint64_t d_boundary{0};
```

Comment:
Looking at the diff, it seems that `d_name` and `d_boundary` could still be `const`?

Reply:
I had some issues there with an earlier version, will revisit.